ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1165,3 +1165,15 @@ admin.message.queue.tab.scheduled = Scheduled
 admin.message.queue.tab.due = Due
 admin.message.queue.tab.muted.future = Will not send
 admin.message.queue.tab.muted.past = Did not send
+username.taken = Username "{{username}}" already taken.
+contact.place.invalid = Contact is not within place.
+contact.type.wrong = Wrong type, contact is not a person.
+username.new.invalid = Failed to validate new username:
+user.not_found = Failed to find user
+user.settings.not_found = Failed to find user settings.
+fields.required = Missing required fields: 
+permissions.required = You do not have permission to modify:
+fields.one.required = One of the following fields are required:
+fields.place.required = Place field is required for offline users
+fields.contact.required = Contact field is required for offline users 
+password.weak.score = The password is too easy to guess. Include a range of types of characters to increase the score.
```

Comment:
Some of the new translations are no longer used. We should only keep the ones that are.